charts/nextcloud/templates/secrets-hooks.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if not (values .Values.nextcloud.hooks | compact | empty) }}
+---
+apiVersion: v1
+kind: Secret
```

Comment:
Can you explain why you chose Secret over a simple ConfigMap?

Reply:
I like to write an script/hook to setup oidc (which contains client-secret).

or other tokens and secrets could be in the scripts

Reply:
(but yes i could change it to ConfigMap and use therefore environment variables)

Reply:
I see. I think this should just be a ConfigMap and then you can use the environment variables for the real secrets.